source/workflow/app.py

```diff
@@ -52,6 +52,11 @@
 else:
     ""
 
+if "ShortUUID" in os.environ:
+    ShortUUID = os.environ["ShortUUID"]
+else:
```

Comment:
Might be clearer to remove this else statement if it isn't doing anything. Not sure what the quotes are for.